content/using.html

```diff
@@ -66,25 +66,28 @@
 
   {% endcall %}
 
-  {% call subsection('Channels')%}
+  {% call subsection('Labels')%}
 
-  Each file within a package may be tagged in one or more channels. A file may also not be tagged in any channels.
-  One common use of channels is to divide a package's files into development, testing, and main channels called 
-  dev, test, and main. This allows package authors to upload files for development or testing purposes without
-  affecting non-development users.
-
-  At [https://**pypi**.anaconda.org/conda](https://pypi.anaconda.org/conda) a channel
-  dropdown lists all the available channels in this user namespace.
+  Each file within a package may be tagged with one or more labels, or not tagged at all. 
```

Comment:
Sean, do you mean that main is no longer the default, it must actually be labeled explicitly?